tests/test_graph_attributes.py

```diff
@@ -0,0 +1,121 @@
+#############################################################################
+# Copyright (c) 2017-present, Opus One Energy Solutions Corporation
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#############################################################################
+
+import networkx
+
+from asciigraf import graph_from_ascii
+
+
+def test_line_lengths():
+    graph = graph_from_ascii("""
+            <13>           <10>
+        n0-------------n1----------n2
+                       |
+                       |  <3>
+                       |
+                       n4
+    """)
+
+    lengths = networkx.get_edge_attributes(graph, "length")
+    assert lengths == {
+        ("n0", "n1"): 13, ("n1", "n2"): 10,
+        ("n1", "n4"): 3,
+    }
+
+
+def test_node_positions():
+    graph = graph_from_ascii(
+        """
+                Node_1---Node2-----
+                                 /
+                               Nald33
+        """)
+    assert graph.node["Node_1"]["position"] == (16, 1)
```

Comment:
We used to leak Point objects here, with this change we'll return a tuple so that nobody can have their stuff break if we modify the Point class.